config.py

```diff
@@ -54,7 +54,17 @@
 
 HIDE_RESTRICTED_COMMANDS = True
 
+
+def make_dir(path):
+    try:
+        os.mkdir(os.path.join(BOT_ROOT, path))
+    except OSError:
+        if os.path.isdir(os.path.join(BOT_ROOT, path)):
```

Comment:
why not do this first before trying to make the directory?

will `os.mkdir` recursively create subdirectories that are missing?

Reply:
I removed the method completely and just did
```
if os.path.isdir(os.path.join(BOT_ROOT, BOT_DATA_DIR)):
    os.mkdir(os.path.join(BOT_ROOT, BOT_DATA_DIR))
```